invenio_app_ils/circulation/mail/messages.py

```diff
@@ -93,3 +93,35 @@ def trigger_template(self):
             if not get_available_item_by_doc_pid(self.loan["document_pid"]):
                 return "request_no_items"
         return self.trigger
+
+
+class OverdueLoanMessage(BlockTemplatedMessage):
+    """Loader for loan overdue messages."""
+
+    default_template = "invenio_app_ils_mail/overdue.html"
+
+    def __init__(self, loan, document_title, patron_email, days_ago,
+                 profile_url, **kwargs):
+        """Create overdue loan message."""
+        sender = current_app.config["MAIL_NOTIFY_SENDER"]
+        bcc = current_app.config["MAIL_NOTIFY_BCC"]
+        cc = current_app.config["MAIL_NOTIFY_CC"]
+
+        template = current_app.config["OVERDUE_LOAN_MAIL_TEMPLATE"] or \
+            self.default_template
+
+        super(OverdueLoanMessage, self).__init__(
+            template=template,
+            ctx=dict(
+                loan=loan,
+                document_title=document_title,
+                patron_email=patron_email,
+                days_ago=days_ago,
+                profile_url=profile_url,
```

Comment:
This was done before, I suggest to reduce the number of fields because the payload of the document can be quite big... Maybe we can instead just select a list of interesting fields?
e.g. `title`, `authors`, `pid`, etc...